### DISCUSSION TITLE
Container instances

### DISCUSSION THREAD

**Discussion by julisubu** [DouglasNeuroInformatics/OpenDataCapture#1052] [Q&A]
we are trying to upload Open Data Capture to Azure container instances, but we have encountered a problem when uploading the mongo database with the configurations that are in the compose, since docker hub does not allow the copy to Azure in free license, we tried provisioning a mongo seus but when we upload the API image it does not allow us to start this container,   I wanted to ask if you can give us options to perform the container instances

**Comment by gdevenyi** [comment]
Hi,

We have developed and tested using docker on Linux, OSX and Windows(WSL). We currently have no testing or experience with any cloud providers which use alternative interfaces for managing docker containers, as such it's hard to offer any support.

The containers we use are all configured and managed via their environment variables so if you have specific issues with getting something working please share the environment you're passing into the contianer as well as all logs it produces. Thanks.

**Reply by julisubu** [reply]
I'm trying to configure the database in sulfur, a mongol cosmos but it doesn't have the rs0 configuration parameter, it doesn't let me put it, I put a connection string and the API fails. I'm decoupling by uploading each independent service, but the database with Docker doesn't upload. Maybe they have documentation of what each environment variable means in the API component

**Reply by joshunrau** [reply]
> I'm trying to configure the database in sulfur, a mongol cosmos but it doesn't have the rs0 configuration parameter, it doesn't let me put it, I put a connection string and the API fails. I'm decoupling by uploading each independent service, but the database with Docker doesn't upload. Maybe they have documentation of what each environment variable means in the API component

Unfortunately, using a replication set is mandatory for Prisma (see https://github.com/prisma/prisma/issues/8266).
